recipes/libcurl/all/conanfile.py

```diff
@@ -326,67 +329,6 @@ def _patch_cmake(self):
             replace_in_file(self, cmakelists,
                                 "include(CurlSymbolHiding)", "")
 
-        # brotli
-        if Version(self.version) < "8.2.0":
-            replace_in_file(self, cmakelists, "find_package(Brotli QUIET)", "find_package(brotli REQUIRED CONFIG)")
-        else:
-            replace_in_file(self, cmakelists, "find_package(Brotli REQUIRED)", "find_package(brotli REQUIRED CONFIG)")
-        replace_in_file(self, cmakelists, "if(BROTLI_FOUND)", "if(brotli_FOUND)")
-        replace_in_file(self, cmakelists, "${BROTLI_LIBRARIES}", "brotli::brotli")
-        replace_in_file(self, cmakelists, "${BROTLI_INCLUDE_DIRS}", "${brotli_INCLUDE_DIRS}")
-
-        # zstd
-        # Use upstream FindZstd.cmake because check_symbol_exists() is called
-        # afterwards and it would fail with zstd_LIBRARIES generated by CMakeDeps
-        replace_in_file(self, cmakelists, "find_package(Zstd REQUIRED)", "find_package(Zstd REQUIRED MODULE)")
-        if Version(self.version) < "8.10.0":
-            replace_in_file(self, os.path.join(self.source_folder, "CMake", "FindZstd.cmake"), "if(UNIX)", "if(0)")
-
-        # c-ares
-        if Version(self.version) < "8.10.0":
-            replace_in_file(self, cmakelists, "find_package(CARES REQUIRED)", "find_package(c-ares REQUIRED CONFIG)")
-            replace_in_file(self, cmakelists, "${CARES_LIBRARY}", "c-ares::cares")
```

Comment:
This seems like it's only ever used in the Find files, so no need to port this to the new CMakeDeps based approach